frontend/src/components/reviews/ProductReview.jsx

```diff
@@ -0,0 +1,84 @@
+import RatingSummary from "./RatingSummary"
+import ReviewForm from "./ReviewForm"
+import Review from "./Review"
+import { useState, useEffect } from "react"
+import "@/styles/reviews/product_review.css"
+import { getReviewsByAmount } from "@/requests/ReviewRequest"
+import Button from "../Button"
+const ProductReview = ({ product }) => {
+    const [reviews, setReviews] = useState([]);
+
+    useEffect(() => {
+        showMoreReviews()
+    }, [])
+
+
+    const showMoreReviews = () => {
+        let index = reviews.length === 0 ? 0 : reviews.length
+        getReviewsByAmount(product._id, index, 3).then((data) => {
+            let newReviews = [...reviews, ...data]
+            setReviews(newReviews)
+        }).catch((err) => {
+            console.log(err)
+        })
```

Comment:
Currently all errors are printed in console, you can be more specific with the error, to improve future debugging:
.then((data) => {
         let newReviews = [...reviews, ...data];
         setReviews(newReviews);
      })
      .catch((err) => {
         console.error("Error fetching reviews:", err);
         // Manejar el error de manera adecuada (mostrar un mensaje al usuario, etc.).
      });